### PR TITLE
POD: stop abusing comment

### PR DESCRIPTION
**Pull request by levitte**
OpenSSL uses some POD directives masquerading as 'comment'
('=for comment' etc).  This is abusive and confusing.  Instead, we use
our own keyword.

    =for openssl whatever

    =begin openssl

    whatever

    =end openssl

(we have never used the multiline form, but might start one day)

##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by richsalz** [comment]
Can this  be put on hold until all my other doc fixes are merged

**Comment by richsalz** [comment]
And also, https://github.com/openssl/openssl/pull/9974 could use the multi-line form.  That PR needs a second review, approval, and merge.

**Review by paulidale — DISMISSED** [review]

**Comment by richsalz** [comment]
Any thought on the multi-line stuff (in #9974) ? SEparate PR for that?

**Comment by paulidale** [comment]
I guess a multi line form would be useful.  I'm not fussed much.

**Comment by levitte** [comment]
Rebased

**Review by t8m — APPROVED** [review]

**Comment by levitte** [comment]
Merged.

bb82531f6592f0e9af28d3502346191a465374a3 POD: stop abusing comment

**Pull request closed without merging**
